com.unity.shadergraph/Documentation~/Instance-ID-Node.md

```diff
@@ -0,0 +1,11 @@
+# Instance ID Node
+
+## Description
+
+Provides access to the **Instance ID** assigned when rendering with GPU instancing.
```

Comment:
"Provides access to the Instance ID that the GPU assigns when you render with GPU instancing."
(assigned is passive voice, so I am trying to fix it here)